diplomarbeit/app.vue

```diff
@@ -1,6 +1,153 @@
 <template>
-  <div>
-    <NuxtRouteAnnouncer />
-    <NuxtWelcome />
+  <div class="bg-sc-white flex flex-col scroll-smooth bg-[url('/assets/img/bg-houses.svg')] bg-no-repeat bg-[center_top_20rem] sm:bg-[url('/assets/img/bg-houses-desktop.svg')] sm:bg-[center_top_18rem] sm:bg-contain">
```

Comment:
```suggestion
  <div class="bg-sc-white flex flex-col scroll-smooth bg-[url('/assets/img/bg-houses.svg')] bg-no-repeat bg-[center_top_20rem] sm:bg-[url('/assets/img/bg-houses-desktop.svg')] sm:bg-[center_top_18rem] sm:bg-contain min-h-[100svh]">
```
wenn man ganz rausscrollt ist der hintergrund dann immer noch da